srtcore/buffer.cpp

```diff
@@ -1745,16 +1745,19 @@ steady_clock::time_point CRcvBuffer::getTsbPdTimeBase(uint32_t timestamp_us)
             /* Exiting wrap check period (if for packet delivery head) */
             m_bTsbPdWrapCheck = false;
             m_tsTsbPdTimeBase += microseconds_from(int64_t(CPacket::MAX_TIMESTAMP) + 1);
-            HLOGC(tslog.Debug, log << "tsbpd wrap period ends - NEW TIME BASE: "
-                   << FormatTime(m_tsTsbPdTimeBase));
+            LOGC(tslog.Debug, log << "tsbpd wrap period ends with ts=" << timestamp_us
+                << " - NEW TIME BASE: " << FormatTime(m_tsTsbPdTimeBase)
+                << " drift: " << m_DriftTracer.drift() << "us");
         }
     }
     // Check if timestamp_us is in the last 30 seconds before reaching the MAX_TIMESTAMP.
     else if (timestamp_us > (CPacket::MAX_TIMESTAMP - TSBPD_WRAP_PERIOD))
     {
         /* Approching wrap around point, start wrap check period (if for packet delivery head) */
         m_bTsbPdWrapCheck = true;
-        HLOGP(tslog.Debug, "tsbpd wrap period begins");
+        LOGC(tslog.Debug, log << "tsbpd wrap period begins with ts=" << timestamp_us
+            << " drift: " << m_DriftTracer.drift() << "us"
+            << ". First packet in buffer: " << describeState(1));
```

Comment:
The call to `describeState` could be a burden for performance for non-heavy logs.

Reply:
Normally it should happen only once in 1,5 hours.